⚠ instance variable @_disable_counter_cache not initialized

amatsuda opened the pull request:
Here's a fix for a trivial Ruby-level warning.

amatsuda committed:
:warning: instance variable @_disable_counter_cache not initialized

BenMorganIO commented on lib/paranoia.rb:
```diff
@@ -174,7 +174,7 @@ def really_destroy!
 
   def each_counter_cached_associations
-    !@_disable_counter_cache && defined?(super) ? super : []
+    !(defined?(@_disable_counter_cache) && @_disable_counter_cache) && defined?(super) ? super : []
```
Merci! I think you can also delete the `defined?(super)` check here as well if you want. We've dropped support for Rails 4.1 and this was meant for that old version.

amatsuda replied:
@BenMorganIO Done!

amatsuda committed:
All currently supported versions of AR defines each_counter_cached_associations

This `defined?(super)` check was introduced for Rails 4.1 support
https://github.com/rubysherpas/paranoia/pull/390#issuecomment-284819747
but we've stopped supporting Rails 4.1 already
https://github.com/rubysherpas/paranoia/pull/393#issuecomment-289227091
